public/js/helpers/meters.js

```diff
@@ -320,10 +314,10 @@ export class ProgressMeter extends Meter {
 
     opts.meterColor = opts.meterColor || '#2970ff'
     opts.meterWidth = opts.meterWidth || 14
-    opts.centralFontSize = opts.centralFontSize || 18
+    opts.centralFontSize = opts.centralFontSize || 20
     opts.successColor = opts.successColor = '#2dd8a3'
     opts.dotSize = opts.dotSize || 3
-    if (opts.showIndicator === undefined) opts.showIndictor = true
+    if (opts.showIndicator === undefined) opts.showIndicator = false
```

Comment:
Fixing same typo as above, but also setting a default of `false`, which I think was the intention anyway.